router-tests/automatic_persisted_queries_test.go

```diff
@@ -36,6 +39,11 @@ func TestAutomaticPersistedQueries(t *testing.T) {
 
 		t.Run("Sha with query works", func(t *testing.T) {
 			testenv.Run(t, &testenv.Config{
+				RouterOptions: []core.Option{
+					// This ensures that no CDN client for persistent operations is created, so we can verify that
+					// APQ alone (without persistent operation support setup) works as expected.
+					core.WithGraphApiToken(""),
```

Comment:
This is required, because otherwise we automatically create a persistent operations client if the graph api token is set: 

https://github.com/wundergraph/cosmo/blob/444fb8dbc79a076c1a12c2a2ba813b91a19fc79d/router/core/router.go#L910-L925